icons/rotaryinternational.svg

```diff
@@ -0,0 +1 @@
303-.099-.059-.222-.059-.355.002l-3.322 1.617c-.116.054-.242.115-.26.223l-.003.03zm5.239-1.86a2.385 2.385 0 0 0 2.384-2.381 2.393 2.393 0 0 0-2.384-2.389 2.384 2.384 0 0 0 0 4.77zm1.008 4.271c.056.044.148.057.29.022 1.589-.381 2.754-1.055 3.89-2.246.067-.078.141-.159.12-.249-.021-.086-.094-.163-.25-.236l-3.33-1.617a.384.384 0 0 0-.35-.006c-.099.062-.158.167-.167.313l-.27 3.686c-.003.121-.017.259.067.333zm4.966-4.297c.099.064.214.142.318.106.07-.022.132-.1.17-.239.467-1.58.467-2.93 0-4.492-.028-.098-.074-.199-.156-.236-.083-.03-.187.005-.337.106l-3.054 2.077c-.118.079-.178.187-.178.3 0 .11.06.22.178.297zm-.673-6.608c.004-.063-.035-.143-.124-.23-1.134-1.2-2.3-1.871-3.898-2.251-.09-.024-.193-.045-.262.016-.072.058-.093.168-.079.334l.264 3.697c.009.14.071.251.165.307.103.059.225.048.353-.014l3.33-1.606c.106-.052.227-.109.25-.223.001-.007.005-.016.001-.03zM3.073 14.352l1.417-.412a.34.34 0 0 0 .232-.351c-.509-2.64.138-4.78 1.685-6.494.156-.112.173-.34.023-.497L5.4 5.562c-.1-.114-.282-.114-.438.045-1.975 2.078-2.972 5.267-2.24 8.527.04.157.164.272.351.218zm17.618.759-1.42-.486c-.075-.027-.269-.027-.37.223-1.148 2.751-3.819 4.634-6.911 4.634a7.471 7.471 0 0 1-6.915-4.677.288.288 0 0 0-.357-.18l-1.413.485c-.154.056-.246.192-.165.388 1.388 3.541 4.839 6.041 8.85 6.041 4.01 0 7.476-2.517 8.868-6.056.073-.185-.015-.317-.167-.372zM19.05 5.592c-.14-.17-.37-.144-.436-.026l-1.027 1.032c-.15.156-.136.385.02.497 1.55 1.715 2.197 3.85 1.69 6.494a.338.338 0 0 0 .232.352l1.42.411c.189.054.31-.061.35-.223.73-3.255-.27-6.455-2.249-8.537zm-3.308-.384.327-.63.882-.41c-.093-.066-.193-.113-.286-.174l-.608.277-.112-.66c-.127-.06-.203-.086-.335-.14l.177.96-.327.621zM14.09 19.484c-.44.097-.779.177-1.215.244l.04.254c.162-.026.31-.045.48-.066l.22 1.22.323-.061-.262-1.217c.166-.041.32-.083.485-.122zm-4.322.214v.082c-.024.202-.16.406-.382.432l.051.704a1.985 1.985 0 0 1-.329-.104l-.01-.662-.299-.128-.24.585c-.099-.033-.196-.064-.291-.11l.306-.712.036-.087.249-.587.677.261c.15.054.22.188.232.326zm-.284 0a.126.126 0 0 0-.078-.1l-.38-.14-.085.24-.03.087-.003.006.345.129c.095.042.187-.042.22-.135.01-.025.02-.056.01-.087zM14.08 4.51c-.121-.035-.211-.05-.344-.088l.245-.837.18-.597c.259.064.505.148.75.228.14.043.214.201.229.369.009.228-.105.475-.335.527l.09.675a20.875 20.875 0 0 0-.308-.126l-.078-.59-.289-.088zm.207-.79.338.11c.093.026.215-.119.215-.245 0-.06-.02-.114-.101-.132a2.657 2.657 0 0 0-.348-.104l-.07.236zm-1.642 16.696.327.793a4.036 4.036 0 0 0-.303.027l-.1-.257-.655.019-.075.268a3.47 3.47 0 0 0-.306-.012l.282-.838.201-.636.01-.02.333-.015.017.04zm-.198.289-.118-.289-.118-.313-.103.313-.101.308zm2-1.348.5 1.428c.098-.034.202-.063.3-.106l-.485-1.422c-.133.045-.194.063-.315.1zM7.185 4.293l-.084-.15c.22-.14.457-.261.685-.384.269-.148.648.211.621.534a.34.34 0 0 1-.054.16l.55.4c-.105.048-.183.09-.306.143l-.481-.351-.266.142.283.467c-.11.07-.176.113-.304.188zm.332-.071.041.07.153.265.317-.167c.03-.014.047-.051.052-.097.007-.144-.097-.338-.247-.247-.117.061-.189.103-.316.176z"/></svg>
```

Comment:
It looks like there are some differences between your source and the original (your version in black over the original in red). Can you try again with the official source?
![image](https://user-images.githubusercontent.com/12817388/130461247-b5d67a6d-a368-4948-aa81-d0a2debb3e08.png)

Reply:
@service-paradis I've updated this icon with the svg found in their website.
Notice the rightmost tooth of the gear in the source svg is not flat headed. Not sure if e should leave it or flatten it.
I did take the liberty to make the wheel 24x24. Originally it was 23.something x 24, but since it is a circle, I think it should be 24 x 24